### PR TITLE
feat: bump graphql construct to stable/v1

### PR DESCRIPTION
**Pull request by alharris-at**
#### Description of changes
* Update CDK construct package name to remove the `alpha` suffix, and use approved naming `@aws-amplify/graphql-api-construct`
* Bump the library version to 1.0.0.
* Update CDK stability from `experimental` to `stable`.
* Update all remaining references in the repo to remove the `alpha` suffix (e.g. e2e tests, README docs, etc.)
* Fix a README error where I was still calling the `definition` property `schema`.

##### CDK / CloudFormation Parameters Changed
N/A

#### Issue #, if available
N/A

#### Description of how you validated changes
Build, and verify .jsii assembly is updated, running e2e tests as well.

#### Checklist
- [x] PR description included
- [x] `yarn test` passes
- [x] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#tests)
- [x] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-e2e-tests/src/__tests__/cdk/backends/base-cdk/app.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@
 import 'source-map-support/register';
 import { App, Stack, Duration } from 'aws-cdk-lib';
 // @ts-ignore
-import { AmplifyGraphqlApi, AmplifyGraphqlDefinition } from '@aws-amplify/graphql-construct-alpha';
+import { AmplifyGraphqlApi, AmplifyGraphqlDefinition } from '@aws-amplify/api-graphql-construct';
```
Comment (dpilch) [inline packages/amplify-e2e-tests/src/__tests__/cdk/backends/base-cdk/app.ts:5]: Should the package name be `@aws-amplify/graphql-api-construct` to align with the import name `AmplifyGraphqlApi`?

Comment (alharris-at) [reply packages/amplify-e2e-tests/src/__tests__/cdk/backends/base-cdk/app.ts:5]: @renebrandel to confirm, but this is approved by leadership specifically.

**Review by renebrandel — DISMISSED** [review]

File: packages/amplify-e2e-tests/src/__tests__/cdk/backends/base-cdk/app.ts (hunk 1 above)
Comment (alharris-at) [reply packages/amplify-e2e-tests/src/__tests__/cdk/backends/base-cdk/app.ts:5]: Yeah, we'll flip to `graphql-api-construct`. I'll update that in the next rev.

**Review by dpilch — APPROVED** [review]

**Review by renebrandel — APPROVED** [review]

**Review by iartemiev — APPROVED** [review]

**Comment by alharris-at** [comment]
All E2Es passing except for the one that's been failing for quite a while.

File: packages/amplify-graphql-api-construct/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@aws-amplify/graphql-construct-alpha",
-  "version": "0.9.0",
+  "name": "@aws-amplify/graphql-api-construct",
+  "version": "1.0.0",
```
Comment (phani-srikar) [inline packages/amplify-graphql-api-construct/package.json:3]: If we update this manually, lerna is going to bump it to `1.1.0` or `1.0.1` depending on the changes.

**Review by phani-srikar — DISMISSED** [review]

**Review by dpilch — APPROVED** [review]

**Review by renebrandel — APPROVED** [review]

**Review by iartemiev — APPROVED** [review]

**Pull request merged**